lib/rubocop/config_loader.rb

```diff
@@ -60,7 +60,7 @@ def create_config(hash, path)
       end
 
       def add_missing_namespaces(path, hash)
-        hash.keys.each do |k|
+        hash.clone.each_key do |k|
           q = Cop::Cop.qualified_cop_name(k, path)
           next if q == k
 
```

Comment:
```Ruby
# Original lib/rubocop/config_loader.rb
def add_missing_namespaces(path, hash)
  hash.keys.each do |k|
    q = Cop::Cop.qualified_cop_name(k, path)
    next if q == k

    hash[q] = hash.delete(k)
  end
end
```

The modified Lint/HashEachMethod cop will catch `hash.keys.each` and correct it to `hash.each_key`. This will work- until it runs `hash[q] = hash.delete(k)`, which will spout `RuntimeError: can't add a new key into hash during iteration`. This is because `lazy iterations` have been implemented in Ruby 1.9, forbidding new keys to be added to the hash that is being iterated over. 

One solution that I have looked up is to clone it before iterating over it. Not a big fan of this though. Any suggestions?

```Ruby
# Modified lib/rubocop/config_loader.rb
def add_missing_namespaces(path, hash)
  hash.clone.each_key do |k|
    q = Cop::Cop.qualified_cop_name(k, path)
    next if q == k

    hash[q] = hash.delete(k)
  end
end
```